src/test.rs

```diff
@@ -158,6 +158,53 @@ fn cross_process_embedded_senders() {
     assert_eq!(received_person, person);
 }
 
+#[cfg(not(any(feature = "force-inprocess", target_os = "windows", target_os = "android", target_os = "ios")))]
```

Comment:
Why are you excluding the `inprocess` back-end? I don't see any obvious reason why the test wouldn't apply there?